src/components/point.ts

```diff
@@ -0,0 +1,23 @@
+import { Component } from "react";
+
+export default class Point extends Component<{}, {}> {
```

Comment:
Might need to make this an SFC. Check the way alert is implemented in any of the FOB repos